truss/truss/config.go

```diff
@@ -0,0 +1,41 @@
+package truss
+
+import (
+	"path/filepath"
+)
+
+// Config defines the inputs to a truss service generation
+type Config struct {
+	// The first path in $GOPATH
+	GOPATH string
+
+	// The path to where .pb.go files protoc-gen-go creates will be written
```

Comment:
I feel defaulting implies that this package will default it to that, which it could do, it just does not do that now. Only main.go knows this to be the case. I could add that comment though, I personally don't mind.